### ISSUE TITLE
how to generate a single ara report for multiple ansible calls in CI

### ISSUE THREAD

**Issue by ssbarnea** [pycontribs/ara#3]
Assuming that you would call ansible 3-10 times during your job execution while are is enabled via `ANSIBLE_CALLBACK_PLUGINS=/PATH-TO/ara/plugins/callbacks`

There are few aspects that need clarification:
* How to avoid cross contamination of the ARA database from ~/.ara when multiple builds are run
* How to generate the unified reports as `ara generate html <ID>` as you may not know the IDs (``ara playbook list``).

**Comment by dmsimard** [comment]
I think we've [discussed this on IRC](http://eavesdrop.openstack.org/irclogs/%23ara/%23ara.2018-04-17.log.html#t2018-04-17T14:58:31) and you managed to figure this out.

Let me know if you have other questions regarding this.

**Comment by ssbarnea** [comment]
Yep this is fixed but I will not close it as I will want to raise a CR that would improve documentation how to use ara a little bit. I plan to write a small guide on how to add ARA to Jenkins jobs.
